cmd2/cmd2.py

```diff
@@ -130,7 +134,9 @@ def __init__(self, completekey: str = 'tab', stdin=None, stdout=None, *,
                  startup_script: str = '', use_ipython: bool = False,
                  allow_cli_args: bool = True, transcript_files: Optional[List[str]] = None,
                  allow_redirection: bool = True, multiline_commands: Optional[List[str]] = None,
-                 terminators: Optional[List[str]] = None, shortcuts: Optional[Dict[str, str]] = None) -> None:
+                 terminators: Optional[List[str]] = None, shortcuts: Optional[Dict[str, str]] = None,
+                 command_sets: Optional[Iterable[CommandSet]] = None,
+                 auto_load_commands: bool = True) -> None:
```

Comment:
We do because, for projects that choose to use CommandSets they will 'just work'. The search for child classes is fast because it's asking Python for child classes that have already been loaded. This should just be a lookup of information that's already loaded in memory.